test/plot-schema.json

```diff
@@ -53564,7 +53597,7 @@
      ]
     },
     "fillcolor": {
-     "description": "Sets the fill color. Defaults to a half-transparent variant of the line color, marker color, or marker line color, whichever is available.",
+     "description": "Sets the fill color. Defaults to a half-transparent variant of the line color, marker color, or marker line color, whichever is available. If fillgradient is specified, fillcolor is ignored except for setting the background color of the hover label, if any.",
```

Comment:
`scattergl` trace. I don't think we want to implement similar feature for `scattergl` so this description shouldn't change.